### DISCUSSION TITLE
Enhancement to Column editing in SQL Editor

### DISCUSSION THREAD

**Discussion by emanc001** [dbeaver/dbeaver#20673] [Q&A] [answered]
Existing column editing in the SQL Editor (Dbeaver community edition) is very basic and each line to be added required a precise left-click.

If one could drag the mouse to position the cursor in columns across multiple lines, it would be great.

As an example, see this convenient feature in notepadd++ 

![notepad_col_edt](https://github.com/dbeaver/dbeaver/assets/140181700/cee2024a-7790-4aba-a99b-1a1afcf74f98)

**Comment by E1izabeth** [comment, marked as answer]
I believe it's already possible in DBeaver.

1. Press `Alt+Shift+A` to enter block selection mode
2. Use arrow keys or mouse left-click dragging to extend the selection
3.  Edit in column mode
4.  Press `Alt+Shift+A` again to leave block selection mode

Is it what are you looking for?

**Comment by emanc001** [comment]
Hi Elizabeth,

Many thanks for your reply.
Yes, this is exactly what I was looking for!
I am pleased to learn of this.

Thanks.

Regards,
Naresh Chauhan
